src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -608,12 +642,18 @@ export class MatrixRTCSession extends TypedEventEmitter<
     }
 
     /**
-     * Get the active focus from the current CallMemberState event
+     * Get the focus in use from a specific specified member.
+     * @param member The member for which to get the active focus. If undefined, the own membership is used.
      * @returns The focus that is currently in use to connect to this session. This is undefined
      * if the client is not connected to this session.
+     * @deprecated use `member.getTransport(session.getOldestMembership())` instead if you want to get the active transport for a specific member.
      */
-    public getActiveFocus(): Focus | undefined {
-        return this.membershipManager?.getActiveFocus();
+    public resolveActiveFocus(member?: CallMembership): Transport | undefined {
```

Comment:
Since I introduced this method during the initial commits to the multi-SFU branch, it's probably fine to delete it rather than deprecating it. If anything what could get kept around and marked as deprecated are `getActiveFocus` and `getFocusInUse`, somehow.